docs/reference/changelog-r2021.md

```diff
@@ -146,6 +146,7 @@ Released on July, 5th, 2021.
     - Fixed incorrect resetting of [PointLight](pointlight.md) nodes ([#3345](https://github.com/cyberbotics/webots/pull/3345)).
     - Fixed incorrect cleaning of [Light](light.md) node ([3374](https://github.com/cyberbotics/webots/pull/3374)).
     - Fixed crash provoked by canceling and switching world in the Guided Tour ([#3376](https://github.com/cyberbotics/webots/pull/3376)).
+    - Fixed incorrect reset of [BallJoint](balljoint.md) nodes ([#3394](https://github.com/cyberbotics/webots/pull/3394)).
```

Comment:
Note that there is already an entry about fixing reset of BallJoint (line 119).
Could please just add the link to the PR to the already existing entry instead of adding a new entry with almost duplicated text?